dvc/fs/fsspec_wrapper.py

```diff
@@ -252,35 +242,36 @@ def ls(self, *args, **kwargs):
 
 
 class CallbackMixin:
```

Comment:
CallbackMixin is now a compat-layer for filesystems that don't support callbacks yet.